integration/log.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cenkalti/backoff"
```

Comment:
Drive-by comment.

@gdbelvin initially used a 3rd party backoff package, but I asked for it to be removed due to licensing concerns. I don't think it was the same package.

`client/backoff` is general purpose, IIRC. It's just a matter of moving it somewhere else.